src/main/java/in/venkat/service/ShowService.java

```diff
@@ -0,0 +1,204 @@
+package in.venkat.service;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import in.venkat.dao.ShowListDao;
+import in.venkat.exceptions.DbException;
+import in.venkat.exceptions.EmptyFieldException;
+import in.venkat.exceptions.InvalidDetailsException;
+import in.venkat.exceptions.InvalidNameException;
+import in.venkat.model.Show;
+import in.venkat.util.Logger;
+import in.venkat.validator.ValidateSearchDetails;
+
+public class ShowService {
+	private ShowService() {
+		/**
+		 * Adding private constructor
+		 */
+	}
+
+	/**
+	 * This method is used to search movies by giving genre and language details
+	 * 
+	 * @param filmGenre
+	 * @param filmLanguage
+	 * @return
+	 * @throws SQLException
+	 * @throws InvalidDetailsException
+	 * @throws InvalidNameException
+	 * @throws EmptyFieldException
+	 * @throws DbException
+	 * @throws Exception
+	 */
+	public static List<Show> searchContents(String filmGenre, String filmLanguage) {
+		boolean isValid;
+		List<Show> search = null;
+
+		try {
+			isValid = ValidateSearchDetails.validateDetails(filmGenre, filmLanguage);
+			if (isValid) {
+				search = searchMoviesByLanguageAndGenre(filmGenre, filmLanguage);
+			}
+		} catch (EmptyFieldException | InvalidNameException | InvalidDetailsException e1) {
+			Logger.log(e1.getMessage());

```

Comment:
Why you didn't throw exception after catching.???

Reply:
fixed this